docs/source/tutorial.rst

```diff
@@ -477,6 +477,14 @@ Running the simulations on a remote machine requires a few things:
   without a password (see `article on password-less ssh
   <http://askubuntu.com/questions/46930/how-can-i-set-up-password-less-ssh-login>`_.
 - the computer should have a working basic Python interpreter.
+- the remote libpython*.so should be shared object for this install python using the command `./configure --enable-shared`       option or use package management system like miniconda.
+- make sure to add python bin and lib location to $PATH and $LD_LIBRARY_PATH in .bashrc respectively.
```

Comment:
Make sure that the python interpreter is in the ``$PATH`` and that the library directory is in ``$LD_LIBRARY_PATH`` -- again this applies only for a Python binary that you have built by yourself.